canonical-web-and-design/charmhub.io
# Remove mock for actions

## Done

Remove mock returns for actions

## QA

- Check out this feature branch
- Make file: `.env.local` with:

`CHARMSTORE_API_URL=https://api.staging.snapcraft.io/`

- Run the site using the command `dotrun`
- http://0.0.0.0:8045/ubuntu-qa/actions
- Ubuntu-qa is the only charm to have actions for now.
- Try other charms actions pages. Should have an empty state
